pyproject.toml

```diff
@@ -1,14 +1,148 @@
-# pyproject.toml
+[project]
+name = 'datacube'
+dynamic = ['version']
+requires-python = ">=3.10"
+readme = "README.rst"
+license = "Apache-2.0"
+description = "An analysis environment for satellite and other earth observation data"
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Science/Research",
+    "Natural Language :: English",
+    "Operating System :: MacOS :: MacOS X",
+    "Operating System :: POSIX",
+    "Operating System :: POSIX :: BSD",
+    "Operating System :: POSIX :: Linux",
+    "Operating System :: Microsoft :: Windows",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Topic :: Scientific/Engineering :: GIS",
+    "Topic :: Scientific/Engineering :: Information Analysis",
+]
+dependencies = [
+        'affine',
```

Comment:
Since the git blame goes out through the window in this commit anyways, should these and the dependency groups be alphabetically sorted?